oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -353,6 +357,14 @@ static int spml_ucx_init(void)
                        -1, EV_PERSIST, mca_spml_ucx_async_cb, NULL);
     }
 
+    if (mca_spml_ucx.symmetric_rkey) {
+        ret = mca_spml_ucx_rkey_store_create(&mca_spml_ucx.rkey_store,
+                                             mca_spml_ucx.ucp_context, 1000);
```

Comment:
maybe make the size configurable?

Reply:
fixed, passing the added parameter `symmetric_rkey_size`